js/src/features/human-resouce/components/EmployeeLocationList.tsx

```diff
@@ -20,15 +18,8 @@ export const EmployeeLocationList: React.FC<Props> = ({
     {employeeLocations.map((item) => (
       <Timeline.Item
         key={item.id}
-        label={`${item.startedAt} - ${item.endedAt}`}
-        color={
-          isWithinInterval(
-            { start: parseISO(item.startedAt), end: parseISO(item.endedAt) },
-            new Date(),
-          )
-            ? "green"
-            : "blue"
-        }
+        label={`${item.startedAt} - ${!!item.endedAt ? item?.endedAt : 'till now'}`}
```

Comment:
Maybe better to use `??` or `||` operator?

Reply:
fix